docs/core/pihole-command.md

```diff
@@ -74,7 +74,7 @@ Help Command    | N/A
 Script Location | [`/opt/pihole/piholeDebug.sh`](https://github.com/pi-hole/pi-hole/blob/master/advanced/Scripts/piholeDebug.sh)
 Example Usage   | [`pihole debug`](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug)
 
-The Pi-hole debugger will attempt to diagnose any issues, and link to an FAQ with instructions as to how an admin can rectify the issue. Once the debugger has finished, the admin has the option to upload the generated log to the [Pi-hole developers](https://github.com/orgs/pi-hole/teams/debug/members), who can help with diagnosing and rectifying persistent issues.
+The Pi-hole debugger will attempt to diagnose any issues, and link to an FAQ with instructions as to how an admin can rectify the issue. Once the debugger has finished, the admin has the option to upload the generated log to the [Pi-hole developers](https://github.com/orgs/pi-hole/people), who can help with diagnosing and rectifying persistent issues.
```

Comment:
I'm unsure if there is another public facing link so I went with what I can access myself

Reply:
For me, there is content in the old link. Furthermore, the people of the formally used link are the ones with access to the debug logs. However, excluding `pralor`, the people listen on both pages are identical.

Reply:
I guess you have content because you are a team member. Maybe the team isn't public or something?

Reply:
`/people` isn't the right link. There's a specific subset of the team members that have access.

Reply:
Yeah, but non-team members don't see the teams... Try visiting the team links in incognito.

Reply:
I did, it asks me for a github login. The team is set to visible. If that team is not visible then just remove the link completely. I don't want to have people listed that are not able to see the logs. We've moved to be an Open Source organization and we may be adding people to the organization that are not active developers but are contributors instead. So listing all the people would not be correct.

Reply:
I removed the link since it's 404'ing for non-team members.